platform/ios/src/MGLMapView.h

```diff
@@ -1113,6 +1113,15 @@ IB_DESIGNABLE
 #pragma mark Overlaying the Map
 
 /**
+ The complete list of overlays associated with the receiver. (read-only)
+
+ The objects in this array must adopt the `MGLOverlay` protocol. If no
+ overlays are associated with the map view, the value of this property is
+ `nil`.
+ */
+@property (nonatomic, readonly, nullable) NS_ARRAY_OF(id <MGLOverlay>) *overlays;
```

Comment:
As of iOS 9, `MKMapView.annotations` and `MKMapView.overlays` have changed to being non-nullable, returning an empty array when the map contains no annotation or no overlay. This would be a good opportunity to follow suit. (Since making the property non-nullable would also change it from optional to non-optional in Swift, we whould need to add a blurb to both changelogs about this change.)